test/e2e/cypress/support/commands.js

```diff
@@ -97,3 +97,111 @@ Cypress.Commands.add('addTagByColumnValue', (columnValue, tagValue) => {
       cy.get('span').contains('Add Tag').type(`${tagValue}{enter}`);
     });
 });
+
+Cypress.Commands.add('requestMockRunnerResult', (result) => {
+  const [webAPIHost, webAPIPort] = [
+    Cypress.env('web_api_host'),
+    Cypress.env('web_api_port'),
+  ];
+
+  const requestResultBody = JSON.stringify({
+    expected_results: result,
+  });
+
+  const headers = {
+    'Content-Type': 'application/json;charset=UTF-8',
+  };
+
+  const url = `http://${webAPIHost}:${webAPIPort}/api/mockrunner/expected_result`;
+  cy.request({
+    method: 'POST',
+    url: url,
+    body: requestResultBody,
+    headers: headers,
+  });
+});
+
+Cypress.Commands.add('requestChecksExecution', (clusterId) => {
+  const [webAPIHost, webAPIPort] = [
+    Cypress.env('web_api_host'),
+    Cypress.env('web_api_port'),
+  ];
+
+  const headers = {
+    'Content-Type': 'application/json;charset=UTF-8',
+  };
+
+  const url = `http://${webAPIHost}:${webAPIPort}/api/clusters/${clusterId}/checks/request_execution`;
+  cy.request({ method: 'POST', url: url, headers: headers });
+});
+
+Cypress.Commands.add('mockStartChecksExecution', (clusterId, executionId) => {
```

Comment:
I am wondering whether we need 
`mockStartChecksExecution` and `mockCompleteChecksExecution`.

In fact I believe we might not need them.

Why you may ask.

The reason is that once we 
- had instructed the `MockRunner` on the results that it should return on next execution `requestMockRunnerResult`
- requested the execution to start `requestChecksExecution`

then the `MockRunner` itself takes care of calling the appropriate callback functions when needed.

See here https://github.com/trento-project/web/blob/main/lib/trento/application/integration/checks/adapter/mock_runner.ex#L88

Reply:
Nice catch! As originally I was trying to go around the mock runner I didn't notice this wasn't needed anymore +1

Reply:
Great!

I don't know if you already considered the fact that the `MockRunner` takes a randomic amount of time (between 2000 and 4000 ms) to return the results.

Not sure if this is handled by the test.

Reply:
In principle, it should be covered by the default wait time of 4000 ms that cypress implements to get elements from the DOM. I didn't find any flakiness at least on my machine but I'll keep an eye on it.

Reply:
I see, let's keep an eye if there's any flakiness. :+1: